### PR TITLE
Move performance tests to accumulo-testing #1200

### PR DESCRIPTION
**Pull request by lbschanno**
Move ManySplitIT as HighSplitCreationPT.
Move BalanceFasterIT as SplitBalancingPT.
Move DeleteTableDuringSplitIT as TableDeletionDuringSplitPT.
Move RollWALPerformanceIT as RollWALPT.

### CODE REVIEW

File: src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    long start = System.currentTimeMillis();
+    client.tableOperations().addSplits(TABLE_NAME, splits);
+    long totalTime = System.currentTimeMillis() - start;
+    double splitsPerSecond = NUM_SPLITS / (totalTime / ONE_SECOND);
+
+    reportBuilder.parameter("splits_per_second", splitsPerSecond,
+        "The average number of splits created per second.");
+    reportBuilder.parameter("meets_performance_threshold",
+        splitsPerSecond > MIN_REQUIRED_SPLITS_PER_SECOND,
+        "Whether or not the average split creation meets or exceeds the minimum number of splits per second.");
```
Comment (Manno15) [inline src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java]: `reportBuilder.parameter` should be replaced with `reportBuilder.result` for the resulting value of the performance test. Results are written in a different section of the json file that gets written.

Comment (Manno15) [reply src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java]: The same should be done for the results of the other tests as well.

Comment (lbschanno) [reply src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java]: @Manno15 thanks for the input, I'll update the code to do so.

Comment (lbschanno) [reply src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java]: Changes made.

**Comment by milleruntime** [comment]
@Manno15 @lbschanno where you able to run these on a cluster using the performance scripts?  I am not sure of the state of the performance framework but I think these are OK to merge.  It is just difficult to tell what you changed after moving over.

**Comment by milleruntime** [comment]
I will go ahead and merge these tests.  There can be follow up work if any are failing or not working.

**Pull request merged**